tests/HoneyBadgerTest.php

```diff
@@ -101,6 +101,57 @@ public function it_accepts_and_sends_context()
         $this->assertEquals(['foo' => 'bar'], $notification['request']['context']);
     }
 
+    /** @test */
+    public function it_accepts_and_sends_multiple_pieces_of_context()
+    {
+        $client = HoneybadgerClient::new([
+            new Response(201),
+        ]);
+
+        $badger = Honeybadger::new([
+            'api_key' => 'asdf',
+            'handlers' => [
+                'exception' => false,
+                'error' => false,
+            ],
+        ], $client->make());
+
+        $badger->context([
+            'foo' => 'bar',
+            'another' => 'context',
+        ]);
+
+        $response = $badger->notify(new Exception('Test exception'));
+
+        $notification = $client->requestBody();
+
+        $this->assertEquals([
+            'foo' => 'bar',
+            'another' => 'context',
+        ], $notification['request']['context']);
+    }
+
+    /** @test */
+    public function context_method_returns_honeybadger_instance()
```

Comment:
Hmmm, I'm in favour of removing this test, as it only verifies the `return $this` at the end of the method. A more useful test, IMO, would be testing that context can be composed (ie `->context(a, b)->context(c, d)` will give `[ => b, c => d]`. You'd also be verifying the chaining at the same time.

Reply:
yeah, you are right.